docs/src/getting_started.ipynb

```diff
@@ -8,7 +8,17 @@
    "source": [
     "# Getting Started with Temporian\n",
     "\n",
-    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/temporian/blob/last-release/docs/src/tutorials/getting_started.ipynb)"
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/temporian/blob/last-release/docs/src/tutorials/getting_started.ipynb)\n",
+    "\n",
+    "This guide will introduce you to the basics of Temporian, including:\n",
+    "- What is an **EventSet** and how to create one from scratch.\n",
+    "- Visualizing input/output data using **EventSet.plot()** and interactive plots.\n",
+    "- Converting back and forth between EventSets and pandas **DataFrames**.\n",
+    "- Transforming the EventSets by using **operators**.\n",
+    "- How operators work when using **indexes**.\n",
+    "- Commonly used operations like **glue**, **resample**, **lag**, moving windows and arithmetics.\n",
```

Comment:
Nit: It is easier to read a list like this one when all the items have the same sentence construction e.g. All sentence start with an infinitive verb;